add.php

```diff
@@ -162,7 +162,7 @@ function submit() {
                      $("#name-input").val("");
                      $("#email-input").val("");
                      $("#id-input").val("");
-                     $("#id-committee").val("None");
+                     $(".committees-checkbox").checked = false;
```

Comment:
I have no idea if this code works, but this section does need to be changed. Feel free to amend this commit if it's inevitably wrong.

Reply:
I realized, we can probably just not reset the committees checkboxes and call it a feature. Anyone entering multiple is likely to be doing so for the same committee

Reply:
Yeah probably, I can take care of that.